src/Adapter/Cart/CartPresenter.php

```diff
@@ -283,7 +286,20 @@ public function present($cart)
             'id_address_delivery' => $cart->id_address_delivery,
             'id_address_invoice' => $cart->id_address_invoice,
             'vouchers' => $this->getTemplateVarVouchers($cart),
-        ];
+            'minimalPurchaseRequired' =>
+                ($this->priceFormatter->convertAmount($productsTotalExcludingTax) < $minimalPurchase) ?
+                sprintf(
+                    $this->translator->trans(
+                        'A minimum shopping cart total of %s (tax excl.) is required to validate your order.
+                        Current cart total is %s (tax excl.).',
+                        [],
+                        'Cart'
+                    ),
+                    $this->priceFormatter->convertAndFormat($minimalPurchase),
+                    $this->priceFormatter->convertAndFormat($productsTotalExcludingTax)
```

Comment:
it can be tricky if we'd like to show only prices with included tax, I remember that it was confusing for people in few stores which I created that minimal price must be provided tax excluded, maybe as it is implemented from scratch we can think about do something with it?